apps/test/solutions/applab/cspunit3.js

```diff
@@ -0,0 +1,179 @@
+/* global $ */
+
+var testUtils = require('../../util/testUtils');
+var TestResults = require('@cdo/apps/constants').TestResults;
+var _ = require('lodash');
+
+var levelDefinition = {
```

Comment:
where did all these params come from -- are they from a certain free play level in csp unit 3? It might be helpful to note that here, so we can see if test failures here repro on the corresponding level.